### ISSUE TITLE
Upgrade to elm 0.17

### ISSUE THREAD

**Issue by Cspeisman** [joshuaclayton/elm-hop-mailboxes-example#1]
Great example @joshuaclayton. Any plan to update this to elm 0.17?

**Comment by joshuaclayton** [comment]
@Cspeisman yep - I'd be holding off to see how others adapted to https://github.com/elm-lang/navigation and what Hop did as well. Now that Hop supports 0.17, I think I'll give this a whirl.

**Comment by Cspeisman** [comment]
👍 thanks so much for the update!

**Comment by joshuaclayton** [comment]
@Cspeisman if you don't mind, I'd actually prefer this stays open until I merge a PR that addresses this. Thanks!

**Comment by Cspeisman** [comment]
Oop! So sorry!
